python/ray/autoscaler/resource_demand_scheduler.py

```diff
@@ -87,9 +87,12 @@ def calculate_node_resources(
 
         def add_node(node_type):
             if node_type not in self.node_types:
-                raise RuntimeError("Missing entry for node_type {} in "
-                                   "available_node_types config: {}".format(
-                                       node_type, self.node_types))
+                logger.warn(
+                    f"Missing entry for node_type {node_type} in "
+                    f"available_node_types config: {self.node_types}. This "
+                    f"node's resources will be ignored. If you are using an "
+                    f"unmanaged node, set the user_node_type tag.")
```

Comment:
This took me some time to understand what you want me to do.
Why not replace it with:
Missing {node_type} entry under `available_node_types` in the cluster config. This node's ....
Also, set the user_node_type tag is not clear, how do I do it?
Can you do `e.g.,  available_node_types[node_type][user_node_type]=XXX` ?